cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/conformity/modified/CgmesConformity1ModifiedConversionTest.java

```diff
@@ -139,7 +139,7 @@ public void microBEInvalidRegulatingControl() {
 
         RatioTapChanger rtc = network.getTwoWindingsTransformer("_e482b89a-fa84-4ea9-8e70-a83d44790957").getRatioTapChanger();
         assertNotNull(rtc);
-        assertFalse(rtc.hasLoadTapChangingCapabilities());
+        assertTrue(rtc.hasLoadTapChangingCapabilities());
```

Comment:
Just for me, could you explain why it this attribute is `true` now when it was `false` before? The regulating control is supposed to be invalid here and the default value for `loadTapChangingCapabilities` should be `false`.

Reply:
In this unitary test the `ltcFlag` = `true` and the regulating control is invalid. 
In the current version under these circumstances the `loadTapChangingCapabilities` is forced to `false`.
In this PR i changed the behaviour allowing the option `ltcFlag` = `true` and regulating control invalid (See reference below) so according to the new code implementation the `loadTapChangingCapabilities` in the unitary test should be `true`.

Implementation Guidelines – CGMES v2.4.15  page 9

ltcFlag = false and TapChangerControl not present

A real and fixed tap that is not controlled but can be moved
under load, e.g. manually. Power flow cannot be set to change
the tap for voltage control during the calculation. Optimal
power flow might have access to these taps and change in order
to optimize. Also State Estimator may estimate the tap position
to find a better solution for the system state.

Reply:
Okay.